src/index.js

```diff
@@ -20,10 +21,51 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+const operations = ['+', '-', '*', '/', '='];
+const state = {
+  score: 0,
+  prevNumber: '',
+  nextNumber: '',
+  nextOperaion: '',
+};
+
 function render() {
+  function onNumberClick(event) {
+    const value = event.target.textContent;
+
+    if (state.prevNumber === '') {
+      state.prevNumber = parseInt(value, 10);
+    } else if (state.nextOperaion === '') {
+      state.prevNumber += value;
+    }
+
+    document.getElementById('score').textContent = state.prevNumber;
+  }
+
+  function onOperationClick(event) {
+    const value = event.target.textContent;
+    state.nextOperaion = value;
+  }
+
   const element = (
     <div>
       <p>간단 계산기</p>
+      <p id="score">{state.score}</p>
+      <p>
+        {
+          numbers.map((number) => (
+            <button type="button" onClick={(event) => onNumberClick(event)}>{number}</button>
```

Comment:
이 상황에서는 event를 직접 가져오는것보다, number를 이용하는게 좋을 것 같습니다~

그리고 이렇게만 해도 event는 넘어갑니다.
```suggestion
            <button type="button" onClick={onNumberClick}>{number}</button>
```